sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/DateExpressionsSuite.scala

```diff
@@ -777,8 +778,6 @@ class DateExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
           checkEvaluation(
             FromUnixTime(Literal(1000L), Literal.create(null, StringType), timeZoneId),
             null)
-          checkEvaluation(
```

Comment:
tests for invalid formats are gathered into this case https://github.com/apache/spark/pull/28650/files#diff-3cae35e69eb5cf9538d1d0947c13d8ebR1165.